src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,33 @@
+package step2;
+
+public class Calculator {
+
+  private final static String SPLIT_SPACE = " ";
+
+  public int calculate(String input) {
+
+    if(isEmpty(input)) {
+      throw new IllegalArgumentException();
+    }
+
+    String[] splitInputArr = input.split(SPLIT_SPACE);
+
+    int result = Integer.parseInt(splitInputArr[0]);
+    for (int i = 1; i < splitInputArr.length; i += 2) {
+      String operator = splitInputArr[i];
+      int number = Integer.parseInt(splitInputArr[i+1]);
+      result = getCalculateResult(operator, result, number);
+    }
+
+    return result;
+  }
+
+  private int getCalculateResult(String operator, int target, int number) {
```

Comment:
```suggestion
  private int calculate(String operator, int target, int number) {
```

1. 메서드명에 get 대신 다른 동사를 쓰면 좀더 읽기 좋은 코드가 됩니다. getter는 get을 쓰는게 좋지만 다른 기능이라면 기능을 간결하게 잘 표현할 수 있는 메서드명이 좋습니다.
2. Result는 메서드 리턴타입으로 이미 표현되고 있으니 생략하는게 좋겠네요.